sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/PocoEntityValueBinder.cs

```diff
@@ -74,9 +73,7 @@ public Task SetValueAsync(object value, CancellationToken cancellationToken)
 
             if (HasChanges(entity))
             {
-                var table = _entityContext.Table;
-                var operation = TableOperation.Replace(entity);
-                return table.ExecuteAsync(operation, cancellationToken);
+                return _entityContext.Table.UpdateEntityAsync(entity, entity.ETag, cancellationToken: cancellationToken);
```

Comment:
For compat reasons, do we want the default Merge behavior, or specify Replace mode?